Automattic/jetpack
# Jetpack Sync :: Add support for additional meta fields for Jetpack Search

Adds additional meta fields for use by Jetpack Search

## Proposed changes:

Adds syncing of following meta fields
- show_notes
- show_notes_preview
- selected_links
- people_mentioned
- guest_name
- guest_bio
- ama_content

### Other information:

- N/A

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No - privacy documentation already covers Sync activity and in particular meta fields.

## Testing instructions:

* Setup a test site with Jetpack Search plan
* Add meta for the above fields to a post or posts
* Verify the meta is synced to the cache site and indexed
